docs/REST API/rest-api.md

```diff
@@ -6995,36 +7123,128 @@ Deletes the session specified by ID. When the user associated with the session n
           }
         ],
         "hosts_count": 1
+      },
+      {
+        "id": 32,
+        "name": "1Password – Password Manager",
+        "versions_count": 1,
+        "source": "chrome_extensions",
+        "browser": "chrome",
+        "release": "",
+        "vendor": "",
+        "arch": "",
+        "generated_cpe": "",
+        "versions": [
+          {
+            "id": 4242,
+            "version": "2.3.7",
+            "vulnerabilities": []
+          }
+        ],
+        "hosts_count": 345
       }
-    ]
+    ],
+    "meta": {
+      "has_next_results": false,
+      "has_previous_results": false
+    }
 }
 ```
 
-### Count software
+### Get software title
 
-`GET /api/v1/fleet/software/count`
+Returns information about the specified software title. By default, `versions` are sorted in descending order by the `hosts_count` field.
 
-#### Parameters
+`GET /api/v1/fleet/software/titles/:id`
 
```

Comment:
See suggestion below for software version — would still include the parameters table to keep the format consistent

Reply:
Thank you for catching this. I agree, we should be consistent. Added parameters table for both software titles and versions endpoints.